src/Symfony/Component/Mime/Header/UnstructuredHeader.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mime\Header;
+
+/**
+ * A Simple MIME Header.
+ *
+ * @final
+ *
+ * @author Chris Corbyn
+ */
+class UnstructuredHeader extends AbstractHeader
```

Comment:
should be final like the other ones

Reply:
It is final, see the `@final` annotation